spec/test/stored_queries/query_dev/test_fulltext_search_dev.py

```diff
@@ -0,0 +1,486 @@
+import os
```

Comment:
is this file needed or just an accidental commit?

Reply:
This file represents weeks of work I put into making the `generic/fulltext_search.yaml` stored query. I was planning on hooking it up with `make`. Its purpose is to connect with the `ci` database and test the stored query against every single current (as of time of coding) species/strain name in the `ncbi_taxon` collection; collect stats; stratify by complicated/simple scinames and all their prefixes and all single alphanumeric characters; and output mean/median execution times. There's also functionality for graphing the execution times depending on certain token criteria.

Reply:
ok, great. I didn't read the contents due to the file name, but it sounds like it just needs renaming. 👍